README.md

```diff
@@ -2045,18 +2045,13 @@ primacron can be time savers.
 
 #### Can I use cluster?
 
-The `cluster` module that ships with Node.js is flawed, seriously flawed.
-Cluster in node < 0.12.0 lets the Operating System decide which worker process
-should receive the request. This results in a not even distribution across
-the workers. If you have 3 workers, it's possible that one is used at 70%,
-another at 20% and the last one at 10%. This is of course not wanted when using
-a cluster. In addition, the load balancing done by the OS is not sticky.
-
-Cluster in node >= 0.12.0 implements a custom round robin algorithm in order to
-fix this un-even distribution of the load across the workers, but it does not
-address the sticky session requirement.
-
-There are also projects like `stick-session` which attempt to implement
+> Note: The following only applies to websocket emulation transformers like
+> socket.io or engine.io. If you are using `ws` or `uws`, there is no need
```

Comment:
s/socket.io/sockjs/, the socket.io transformer is gone. I would also add faye along ws and uws.